integration-test.sh

```diff
@@ -395,6 +395,54 @@ function test_proxy_connection {
     restore_rally_log
 }
 
+function docker_compose_dev {
+    if [[ "$1" == "up" ]]; then
+        docker-compose -f docker/docker-compose-tests.yml up --abort-on-container-exit
```

Comment:
Should we mention `docker` and `docker-compose` as [prerequisites for development in our docs](https://esrally.readthedocs.io/en/stable/developing.html#prerequisites)?